packages/create/src/core.js

```diff
@@ -99,18 +96,13 @@ function _install(command = 'npm', options) {
   });
 }
 
-export async function installNpm(where) {
+export async function installNpm(where, packageManagerChoice) {
```

Comment:
can we also move the "decision" to the generator so this is
```js
export async function installNpm(where, command) {
```

I think that's easier to understand

Reply:
Not sure if I understood what you meant fully with move the "decision" but I changed it

Reply:
that was exactly what I had in mind 💪